integration-test/src/test/resources/trino-ci-testset/testsets/jdbc-postgresql/00004_query_pushdown.txt

```diff
@@ -0,0 +1,45 @@
+CREATE SCHEMA
+
+USE
+
+CREATE TABLE
+
+CREATE TABLE
+
+INSERT: 1500 rows
+
+INSERT: 15000 rows
+
+"Trino version: %
+%
+    └─ TableScan[table = test.gt_postgresql:gt_db1.customer gt_db1.customer limit=10 columns=[custkey:bigint:int8]]
+           Layout: [custkey:bigint]
+%
+"
+
+"Trino version: %
+%
+    └─ TableScan[table = test.gt_postgresql:gt_db1.customer gt_db1.customer constraints=[ParameterizedExpression[expression=(""phone"") LIKE (?), parameters=[QueryParameter{jdbcType=Optional.empty, type=varchar(6), value=Optional[Slice{base=[B@%, baseOffset=0, length=6}]}]]] limit=10]
```

Comment:
How can we know the impact of a condition that has been pushed down?

Reply:
If predicate pushdown for a specific clause is succesful, the EXPLAIN plan for the query does not include a ScanFilterProject operation for that clause.

https://trino.io/docs/current/optimizer/pushdown.html#predicate-pushdown

Reply:
I see.